[2021-02-23 14:56:32,035] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_2 2021-02-21T00:00:00+00:00 [queued]>
[2021-02-23 14:56:32,506] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_2 2021-02-21T00:00:00+00:00 [queued]>
[2021-02-23 14:56:32,667] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-02-23 14:56:32,703] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-02-23 14:56:32,809] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-02-23 14:56:33,358] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): runme_2> on 2021-02-21T00:00:00+00:00
[2021-02-23 14:56:33,981] {standard_task_runner.py:52} INFO - Started process 62 to run task
[2021-02-23 14:56:34,466] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'runme_2', '2021-02-21T00:00:00+00:00', '--job-id', '8', '--pool', 'default_pool', '--raw', '--subdir', '/home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmpx7bbiec8', '--error-file', '/tmp/tmp_iwh5c2t']
[2021-02-23 14:56:34,526] {standard_task_runner.py:77} INFO - Job 8: Subtask runme_2
[2021-02-23 14:56:38,059] {logging_mixin.py:104} INFO - Running <TaskInstance: example_bash_operator.runme_2 2021-02-21T00:00:00+00:00 [running]> on host 0513e3a4eab6
[2021-02-23 14:56:45,682] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=runme_2
AIRFLOW_CTX_EXECUTION_DATE=2021-02-21T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-02-21T00:00:00+00:00
[2021-02-23 14:56:45,744] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-02-23 14:56:45,762] {bash.py:158} INFO - Running command: echo "example_bash_operator__runme_2__20210221" && sleep 1
[2021-02-23 14:56:46,202] {bash.py:169} INFO - Output:
[2021-02-23 14:56:46,345] {bash.py:173} INFO - example_bash_operator__runme_2__20210221
[2021-02-23 14:56:47,400] {bash.py:177} INFO - Command exited with return code 0
[2021-02-23 14:56:47,962] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=runme_2, execution_date=20210221T000000, start_date=20210223T145632, end_date=20210223T145647
[2021-02-23 14:56:49,157] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-02-23 14:56:49,569] {local_task_job.py:146} INFO - Task exited with return code 0
